modules/gob-web/app.tsx

```diff
@@ -35,20 +33,20 @@ let GlobalStyle = createGlobalStyle`
     }
 
     menu {
-        padding: 0;
-        margin: 0;
+        padding: 0,
+        margin: 0,
```

Comment:
More css in template strings that had their semicolons changed to commas. Please fix all of these.